geonotebook/utils.py

```diff
@@ -13,3 +13,6 @@ def get_kernel_id(kernel):
     connection_file_path = ipykernel.get_connection_file()
     connection_file = os.path.basename(connection_file_path)
     return connection_file.split('-', 1)[1].split('.')[0]
+
+def transform_coordinates(source_srs, target_srs, x, y):
```

Comment:
also does it makes sense to have it (x, y, target_srs) and source_srs arg if not provided can be assumed to wgs84?